src/params/strinct.js

```diff
@@ -0,0 +1,26 @@
+const Str = require('./str');
+
+const rejectedStrings = [
+  '',
+  'undefined',
+  'undef',
+  'null',
+  'NULL'
```

Comment:
add more as discussed

Reply:
Added more rejected strings from
 https://github.com/minimaxir/big-list-of-naughty-strings/blob/master/naughtystrings/internal/resource.go